### PR TITLE
Add paywallViewControlleSizeDidChange to PaywallProxy

### PR DESCRIPTION
**Pull request by vegaro**
Depends on https://github.com/RevenueCat/purchases-ios/pull/3563

### CODE REVIEW

File: ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift [hunk 1]
```diff
@@ -195,6 +198,11 @@ extension PaywallProxy: PaywallViewControllerDelegate {
         guard let (paywallResultHandler, result) = self.resultByVC.removeValue(forKey: controller) else { return }
         paywallResultHandler(result.name)
     }
+
+    public func paywallViewControlleSizeDidChange(_ size: CGSize) {
```
Comment (NachoSoto) [inline ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift:202]: This will need the new name too.

**Review by NachoSoto — APPROVED** [review]

**Review by NachoSoto — APPROVED** [review]

**Comment by vegaro** [comment]
holding merging until new iOS version is out

**Comment by NachoSoto** [comment]
Merged: https://github.com/RevenueCat/purchases-hybrid-common/pull/645

**Comment by NachoSoto** [comment]
Rebased.

**Pull request merged**